### DISCUSSION TITLE
[BUG] Issue with Get-PnpListItem date changes

### DISCUSSION THREAD

**Discussion by Johnh42** [pnp/powershell#2557] [General]
I am not certain the issue is related to the module and wanted to bring it up here for review. when trying to change the date of and event with command below you cant just change the start date or the end date both have to be changed together otherwise it updates the other time value with a null entry. 
so if I run the following: 
``` Powershell
Set-PnPListItem -List "Maintenance" -Identity 16 -Values @{"EndDate"="10/31/2022 11:00:00 PM"} -UpdateType UpdateOverwriteVersion
```
The value updates for end date but also makes the start date null but if I run the same thing but also include the startdate it of course has no issues.
``` Powershell
Set-PnPListItem -List "Maintenance" -Identity 16 -Values @{"EventDate"="10/31/2022 10:00:00 PM"; "EndDate"="10/31/2022 11:00:00 PM"} -UpdateType UpdateOverwriteVersion
```

### What is the version of the Cmdlet module you are running?
version 1.8.0

### Which operating system/environment are you running PnP PowerShell on?
- [x] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by veronicageek** [comment]
@Johnh42 - Can you please first update to `1.11` which is the latest major, and also try with a nightly release (i.e.: `1.11.124`) just to make sure. Thanks

**Comment by gautamdsheth** [comment]
Hi, moving this to discussion as it is not an issue with module but something to do with your custom code.
